src/Resources/laravel-debugbar.css

```diff
@@ -266,19 +271,20 @@ a.phpdebugbar-restore-btn {
 }
 
 div.phpdebugbar-header > div > * {
+    padding: 5px 8px;
     font-size: 13px;
-    padding: 5px;
+    height: 20px;
```

Comment:
This fixed height gave me problems before, is this required?

Reply:
Changes applied in the base package through this merged PR https://github.com/maximebf/php-debugbar/pull/647/files should make sure it works well.

<img width="439" alt="Screenshot 2024-04-05 at 14 05 05" src="https://github.com/barryvdh/laravel-debugbar/assets/18613261/11916f37-75de-4bf7-91c5-afb1d3a802a3">

The height of 16px is now being set to 20px as the minimum height of `div.phpdebugbar-header` in this package is  30px, compared to a min-height of 26px in the base package.

![side-by-side](https://github.com/barryvdh/laravel-debugbar/assets/18613261/c19cbb7e-e764-426f-af0e-74984bc7c8c5)